CHANGELOG.md

```diff
@@ -1,40 +0,0 @@
-# <a name="changelog"></a>CHANGE LOG
```

Comment:
I misunderstood your comment - I can add the CHANGELOG back, though I think it should be a clean slate - the changes that were in there before were not CivicActions related